### ISSUE TITLE
Rule suggestion: duplicate list markers

### ISSUE THREAD

**Issue by chrisgrieser** [rvben/rumdl#227]
Something that frequently occurs for me is accidentally having duplicate list markers. 

This happens for instance when you copy a list item, go to the next line, and the editor automatically continues the list marker. When you then paste the copied list item, you end up with two list markers:

```
- a line of text
- - a line of text
```

They are easy to overlook, thus warranting a diagnostic. And I think they are a great case for something that could just be autofixed away.

**Comment by rvben** [comment]
Hi @chrisgrieser, thanks for all the issues and ideas! You're really making a difference in making rumdl better for everyone. This suggestion too is an interesting one, and I have to check if there are valid use cases of this to prevent the risk of false positives.

**Comment by chrisgrieser** [comment]
Thank you!

rumdl addresses many issues I had with Markdown tooling, so I was happy to discover it. I also instantly added support for it across the parts of the neovim [ecosystem](https://github.com/mason-org/mason-registry/pull/12811) I have [control over ](https://github.com/chrisgrieser/nvim-rulebook/commit/407d0134a3e16cd05e85706ea02e7a46507a295e):)

**Comment by rvben** [comment]
Added MD069 (no-duplicate-list-markers) which detects exactly this pattern:

- Detects `- - text`, `* * text`, `+ + text`, and mixed markers like `- * text`
- Auto-fix removes the first marker: `- - text` → `- text`
- Correctly ignores CLI flags (`- --verbose`), emphasis (`* *italic*`), and properly indented nested lists

Lands in v0.0.202! Thanks for this great suggestion. 🎉
